commands-yml/commands/device/clipboard/get-clipboard.yml

```diff
@@ -5,6 +5,8 @@ description:
   >
     Get the content of the system clipboard
 
+    For iOS real device, it might work when WebDriverAgentRunner is in [foreground](https://github.com/appium/WebDriverAgent/issues/330).
```

Comment:
For simulator as well. I would rather say. 

Apple security preferences require the `WebDriverAgentRunner` application to be in foreground in order to be able to receive the system clipboard content. Otherwise an empty string is always returned. Consider using ... commands (links to activateApp, backgroundApp) to change the foreground application.

Reply:
I tried now the get_clipboard on a simulator. Then, it worked without foreground stuff.